wolfssl/wolfcrypt/settings.h

```diff
@@ -1318,6 +1325,36 @@ extern void uITRON4_free(void *p) ;
     #define GCM_TABLE
 #endif
 
+#if defined(WOLFSSL_MAXQ1065) || defined(WOLFSSL_MAXQ108X)
+
+    #define MAXQ10XX_MODULE_INIT
+
+    #define HAVE_PK_CALLBACKS
+    #define WOLFSSL_STATIC_PSK
+
+    #define WOLFSSL_USER_MUTEX
```

Comment:
Don't force `WOLFSSL_USER_MUTEX`, just gate this on `MAXQ10XX_MUTEX`. Note their implementation uses pthreads and shared mutex to protect concurrent access between applications. It only works on Linux.

Reply:
> just gate this on MAXQ10XX_MUTEX.

...but below `MAXQ10XX_MUTEX` is gated on `WOLFSSL_USER_MUTEX` so that doesn't make sense to me.  Am I missing something?

Reply:
Just odd logic, but I get it. The `WOLFSSL_USER_MUTEX` is an existing macro.
My concern is ensuring the logic for `MAXQ10XX_MUTEX` is not on by default. It is not portable.

Reply:
I think I get it. If I describe it I'll confuse myself so let me try something.....stay tuned.

Reply:
Fixed with eebb13877b32a7a1324041bf8ecaa922bd7d4693